src/Chapter1.hs

```diff
@@ -632,9 +646,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
-
+sumLast2 :: Int -> Int
+sumLast2 n = last + penultimate  
+  where
+    getLast     = mod n 100
+    last        = mod getLast 10  
+    penultimate = div getLast 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod m 10
div m 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod m 10
```

You can see how we could pattern match on the pair 🙂